eosio.unregd/ram/exchange_state.cpp

```diff
@@ -0,0 +1,82 @@
+#include "exchange_state.hpp"
+
+namespace eosio {
+   asset exchange_state::convert_to_exchange( connector& c, asset in ) {
+
+      real_type R(supply.amount);
+      real_type C(c.balance.amount+in.amount);
+      real_type F(c.weight/1000.0);
```

Comment:
Although it shouldn't make much of a difference as long as the connector weights for both connectors are equal, the `1000.0` factor here is no longer consistent with what is used in the eosio.system contract since v1.1.0. So to reduce errors in what this contract estimates and what the actual system contract calculates, this line should probably be changed to `real_type F(c.weight);`.